python/pyarrow/_dataset_parquet.pyx

```diff
@@ -327,6 +331,7 @@ cdef class ParquetFileFragment(FileFragment):
     @property
     def metadata(self):
         self.ensure_complete_metadata()
+
```

Comment:
Small nitpick: but it's best to avoid such (stylistic) changes in code that is otherwise not touched